openid-caep-specification-1_0.md

```diff
@@ -691,6 +706,56 @@ NOTE: The event type URI is wrapped, the backslash is the continuation character
 ~~~
 {: #device-compliance-change-examples-out-of-compliance title="Example: Device No Longer Compliant - Complex Subject + optional claims"}
 
+## Session Established {#session-established}
+Event Type URI:
+
+`https://schemas.openid.net/secevent/caep/event-type/session-established`
+
+The Session Established event signifies that the Transmitter has created a new session for a user for use with the Transmitter. Receivers may use this information for a number of reasons, including:
+
+* A service acting as a Transmitter can close the loop with the IdP after a user has been federated from the IdP
+* An IdP can detect unintended logins
+* A Receiver can establish an inventory of user sessions
+
+### Event Specific Claims {#session-established-event-specific-claims}
+The following optional claims MAY be included in the Session Established event:
+
+ip
+: The IP address of the user as observed by the Transmitter (**NOTE**, this can be different from the one observed by the Receiver for the same user because of network translation)
+
+fp_ua
```

Comment:
Do we also need an array to indicate which attributes contributed to the fingerprint? This will help create a contract between Tx and Rx on how to arrive at the fingerprint.

Reply:
Can this be a part of the metadata? I don't know if we want to include this information in every event.

Reply:
Does the Rx need to know anything about how to arrive at the fingerprint? What is the intended meaning of this value (see my comment just below)?

Reply:
I don't think there should be a contract between the Transmitter and Receiver about this value. The Receiver should treat it as an opaque value that can be used to correlate events from the Transmitter, but nothing more.

Reply:
Agreed.